### PR TITLE
get latest keep uris

### PR DESCRIPTION
**Pull request by yingjieMiao**
@eishay  @raymondng

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/model/KeepRepo.scala [hunk 1]
```diff
@@ -336,4 +337,13 @@ class KeepRepoImpl @Inject() (
     val oldest = for { bookmark <- bookmarks if bookmark.createdAt <= min } yield bookmark
     oldest.sortBy(_.createdAt asc).firstOption
   }
+
+  def getLatestKeepsURIByUser(userId: Id[User], limit: Int, includePrivate: Boolean = false)(implicit session: RSession): Seq[Id[NormalizedURI]] = {
+    import StaticQuery.interpolation
+
+    val sql = if (includePrivate) sql"select uri_Id from bookmark where state = '#${KeepStates.ACTIVE}' and user_id=${userId} order by created_at DESC limit ${limit}"
```
Comment (raymondng) [inline kifi-backend/modules/shoebox/app/com/keepit/model/KeepRepo.scala:344]: I've nothing against it but am curious: is there a specific reason you go with static query here?

Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/model/KeepRepo.scala:344]: not really. just make the sorting and limiting more explicit.

Comment (yingjieMiao) [inline kifi-backend/modules/shoebox/app/com/keepit/model/KeepRepo.scala:344]: btw, most queries in keepReop are in this style now.

File: kifi-backend/modules/shoebox/test/com/keepit/model/KeepTest.scala [hunk 2]
```diff
@@ -205,5 +205,40 @@ class KeepTest extends Specification with ShoeboxTestInjector {
         }
       }
     }
+
+    "get latest keeps uri by user" in {
```
Comment (raymondng) [inline kifi-backend/modules/shoebox/test/com/keepit/model/KeepTest.scala:209]: Nice :+1:

File: kifi-backend/modules/shoebox/app/com/keepit/model/KeepRepo.scala (hunk 1 above)
Comment (raymondng) [inline kifi-backend/modules/shoebox/app/com/keepit/model/KeepRepo.scala:344]: Np. Just wondering if you noticed anything about performance overhead with the slick generated queries. I think performance would be a (very) good reason to go with plain static query.

**Comment by raymondng** [comment]
lgtm

**Pull request merged**